### PR TITLE
Add two fields to user EmailOverride and Location

### PR DESCRIPTION
**Pull request by Baggerone**

### CODE REVIEW

File: application/migrations/20210714203100_create_users.up.fizz [hunk 1]
```diff
@@ -1,10 +1,12 @@
 create_table("users") {
 	t.Column("id", "uuid", {primary: true})
 	t.Column("email", "string", {})
+	t.Column("email_override", "string", {})
```
Comment (Schparky) [inline application/migrations/20210714203100_create_users.up.fizz:4]:
```suggestion
	t.Column("email_override", "string", {"default": ""})
```

File: application/migrations/20210714203100_create_users.up.fizz [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
 	t.Column("first_name", "string", {})
 	t.Column("last_name", "string", {})
 	t.Column("is_blocked", "bool", {})
 	t.Column("last_login_utc", "timestamp", {})
+	t.Column("location", "string", {})
```
Comment (Schparky) [inline application/migrations/20210714203100_create_users.up.fizz:9]:
```suggestion
	t.Column("location", "string", {"default": ""})
```

**Review by Schparky — APPROVED** [review]
Should `Location` be required at the database level?

File: application/migrations/20210714203100_create_users.up.fizz (hunk 1 above)
Comment (Baggerone) [reply application/migrations/20210714203100_create_users.up.fizz:4]: :+1:

File: application/migrations/20210714203100_create_users.up.fizz (hunk 2 above)
Comment (Baggerone) [reply application/migrations/20210714203100_create_users.up.fizz:9]: :+1:

**Review by jason-jackson — APPROVED** [review]
Generally I think new migration is more kosher. Also, do we have a way to tell if EmailOverride is present, but not selected (for example, if they fill it in but change back to old email), or do we expect UI handle that? That is, should APIs have another field for selection?

**Pull request merged**